src/main/java/com/android/tools/build/bundletool/commands/BuildApksCommand.java

```diff
@@ -92,6 +90,7 @@ public boolean isAnySystemMode() {
   private static final Flag<Path> BUNDLE_LOCATION_FLAG = Flag.path("bundle");
   private static final Flag<Path> OUTPUT_FILE_FLAG = Flag.path("output");
   private static final Flag<Boolean> OVERWRITE_OUTPUT_FLAG = Flag.booleanFlag("overwrite");
+  private static final Flag<Boolean> DEBUGGABLE_FLAG = Flag.booleanFlag("debuggable");
```

Comment:
Can you add also documentation of the flag? See the method help() in this class.